### DISCUSSION TITLE
A conflict about `section`

### DISCUSSION THREAD

**Discussion by AuroraTea** [mdn/mdn-community#479] [Content]
> main content: \<main\>, with various content subsections represented by \<article\>, \<section\>, and \<div\> elements.

In the [MDN Document_and_website_structure](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#html_for_structuring_content)

> \<section\> is similar to \<article\>, but it is more for grouping together a single part of the page that constitutes one single piece of functionality (e.g., a mini map, or a set of article headlines and summaries), or a theme. It's considered best practice to begin each section with a heading; also note that you can break \<article\>s up into different \<section\>s, or \<section\>s up into different \<article\>s, depending on the context.

In the [MDN Document_and_website_structure](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#html_layout_elements_in_more_detail)

> Sectioning content, a subset of flow content, creates a section in the current outline defining the scope of \<header\> elements, \<footer\> elements, and heading content.
>
> Elements belonging to this category are \<article\>, \<aside\>, \<nav\>, and \<section\>.

In the [MDN Content_categories](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#sectioning_content).

> If you are only using the element as a styling wrapper, use a `div` instead.

In the [MDN section](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#usage_notes).

However, there are many places that do not follow this specification, such as [Element Plus container](https://element-plus.gitee.io/zh-CN/component/container.html). No criticism of the component library is intended here, it's just that this is how it's used in [MDN grid-template](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template#examples) as well.

```html
<section id="page">
  <header>Header</header>
  <nav>Navigation</nav>
  <main>Main area</main>
  <footer>Footer</footer>
</section>
```

Apparently, there is a contradiction in terms in MDN.  It should be standardized in the same document, though it's not necessary to make this distinction in modern front-end work.

**Comment by pransh15** [comment]
Hi @AuroraTea 👋 

Thank you so much for starting this discussion. I see that you have kicked off the process and merged the changes in https://github.com/mdn/content/pull/31559

Thanks again for helping us make MDN better! 💜 

Pranshu Khanna
Community Manager, MDN
